src/AccessibilityInsights.SharedUx/packages.config

```diff
@@ -1,20 +1,20 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Axe.Windows" version="0.9.1-prerelease" targetFramework="net471" />
-  <package id="Microsoft.CodeAnalysis.FxCopAnalyzers" version="2.9.8" targetFramework="net471" developmentDependency="true" />
-  <package id="Microsoft.CodeAnalysis.VersionCheckAnalyzer" version="2.9.8" targetFramework="net471" developmentDependency="true" />
-  <package id="Microsoft.CodeQuality.Analyzers" version="2.9.8" targetFramework="net471" developmentDependency="true" />
-  <package id="Microsoft.NetCore.Analyzers" version="2.9.8" targetFramework="net471" developmentDependency="true" />
-  <package id="Microsoft.NetFramework.Analyzers" version="2.9.8" targetFramework="net471" developmentDependency="true" />
-  <package id="Microsoft.VisualStudioEng.MicroBuild.Core" version="0.4.1" targetFramework="net471" developmentDependency="true" />
-  <package id="Microsoft.Win32.Registry" version="4.7.0" targetFramework="net471" />
-  <package id="Newtonsoft.Json" version="12.0.3" targetFramework="net471" />
-  <package id="System.Collections.Immutable" version="1.5.0" targetFramework="net471" />
-  <package id="System.Drawing.Common" version="4.7.0" targetFramework="net471" />
-  <package id="System.IO.FileSystem.Primitives" version="4.3.0" targetFramework="net471" />
-  <package id="System.IO.Packaging" version="4.7.0" targetFramework="net471" />
-  <package id="System.Security.AccessControl" version="4.7.0" targetFramework="net471" />
-  <package id="System.Security.Principal.Windows" version="4.7.0" targetFramework="net471" />
-  <package id="System.Windows.Interactivity.WPF" version="2.0.20525" targetFramework="net471" />
-  <package id="Text.Analyzers" version="2.6.4" targetFramework="net471" developmentDependency="true" />
+  <package id="Axe.Windows" version="0.9.1-prerelease" targetFramework="net472" />
+  <package id="Microsoft.CodeAnalysis.FxCopAnalyzers" version="2.9.8" targetFramework="net472" developmentDependency="true" />
+  <package id="Microsoft.CodeAnalysis.VersionCheckAnalyzer" version="2.9.8" targetFramework="net472" developmentDependency="true" />
+  <package id="Microsoft.CodeQuality.Analyzers" version="2.9.8" targetFramework="net472" developmentDependency="true" />
+  <package id="Microsoft.NetCore.Analyzers" version="2.9.8" targetFramework="net472" developmentDependency="true" />
+  <package id="Microsoft.NetFramework.Analyzers" version="2.9.8" targetFramework="net472" developmentDependency="true" />
+  <package id="Microsoft.VisualStudioEng.MicroBuild.Core" version="0.4.1" targetFramework="net472" developmentDependency="true" />
+  <package id="Microsoft.Win32.Registry" version="4.7.0" targetFramework="net472" requireReinstallation="true" />
```

Comment:
seems reasonable to have requireReinstallation here based on https://docs.microsoft.com/en-us/nuget/consume-packages/reinstalling-and-updating-packages#considerations